functions/go/set-image/image_transformer_test.go

```diff
@@ -30,15 +30,16 @@ func runImageTransformerResults(input, config string) (*fn.ResourceList, error)
 	}
 	rl.FunctionConfig = functionConfig
 	in, _ := rl.ToYAML()
-	out, err := fn.Run(fn.ResourceListProcessorFunc(setImageTags), in)
+	out, err := fn.Run(fn.ResourceListProcessorFunc(SetImageTagSDK), in)
 	if err != nil {
 		return nil, err
 	}
 	rl, _ = fn.ParseResourceList(out)
 
+	rl.Results = nil
```

Comment:
The original method keeps an internal struct `setImageResults` to store all the results, then its `LogResult` method convert the internal struct to be `fn.Results` when the function finishes. For me I just directly append the results while updating the image, as I did in `set-labels`. So unit test differs here, as I reset my Results to nil.